32blit-sdl/File.cpp

```diff
@@ -182,7 +182,13 @@ bool rename_file(const std::string &old_name, const std::string &new_name) {
 }
 
 bool remove_file(const std::string &path) {
-  return remove((basePath + path).c_str()) == 0;
+  // check if the path is under the save path and do not add basePath if this is the case
```

Comment:
This should be unnecessary, since the generic "remove_file" probably still wants to be able to delete things from / (firmware/launcher updates for example)